flash/core/trainer.py

```diff
@@ -171,10 +168,8 @@ def predict(
         output: Union[Output, str] = None,
         **kwargs,
     ):
-        r"""
-        Run inference on your data.
-        This will call the model forward function to compute predictions. Useful to perform distributed
-        and batched predictions. Logging is disabled in the prediction hooks.
+        r"""Run inference on your data. This will call the model forward function to compute predictions. Useful to
```

Comment:
```suggestion
        r"""Run inference on your data.
        
         This will call the model forward function to compute predictions. Useful to
```